src/Faker/Provider/ru_RU/Person.php

```diff
@@ -154,4 +154,28 @@ public function middleName($gender = null)
             static::GENDER_FEMALE,
         )));
     }
+
+    /**
+     * Return last name for the specified gender.
+     *
+     * @access public
+     * @param string|null $gender A gender of the last name should be generated
+     *     for. If the argument is skipped a random gender will be used.
+     * @return string Last name
+     */
+    public function lastName($gender = null)
+    {
+        $lastName = static::randomElement(static::$lastName);
+
+        if (static::GENDER_FEMALE === $gender) {
+            return $lastName . 'a';
+        } elseif (static::GENDER_MALE === $gender) {
+            return $lastName;
+        }
+
+        return static::lastName(static::randomElement(array(
```

Comment:
this won't work: `Person::lastName()` isn't a static function. Also, I'm bot a big fan of using recursion here - I think it's overkill. Can you revert you last commit? Or perhaps you meant `Person::$lastName`?